scripts/batch_file_archiver/batch_file_archiver.py

```diff
@@ -43,11 +45,24 @@ def split_archive_jobs(archive_jobs: Dict) -> List[ArchiveJob]:
     @staticmethod
     def write_job_json(archive_job: ArchiveJob) -> str:
         with open(archive_job.job_file_path, "w") as job_dict_file:
-            json.dump(archive_job.job_dict, job_dict_file)
+            json.dump(archive_job.job_dict, job_dict_file, indent=4, separators=(',', ': '))
             return job_dict_file.name
 
     def bsub_command(self, archive_job: ArchiveJob) -> str:
-        return f' bsub -M 64000 "{self.singularity_command(archive_job)}"'
+        job_memory = self.get_job_memory(archive_job)
+        return f' bsub -M {str(job_memory)} "{self.singularity_command(archive_job)}"'
+
+    def get_job_memory(self, archive_job: ArchiveJob) -> int:
+        s3 = boto3.client('s3',
+                          aws_access_key_id=self.aws_access_key,
+                          aws_secret_access_key=self.aws_key_secret
+                          )
+        job_urls = [file_input['cloud_url'] for file_input in
+                    archive_job.job_dict.get('jobs')[0]['conversion']['inputs']]
```

Comment:
@ESapenaVentura Just notice, some jobs don't have conversion right? This line would throw an error in the case. 

When a job doesn't have conversions, should we set the memory to something low? I'm guessing that the jobs that just require download/upload use little memroy

Reply:
oh yeah. Didn't think about that case scenario! Yep, some jobs won't have conversion. an 'if' statement returning a job memory of 1000 (default in LSF) should be alright